sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIClientBuilder.java

```diff
@@ -292,8 +291,17 @@ private HttpPipeline createHttpPipeline() {
         HttpPolicyProviders.addBeforeRetryPolicies(policies);
         policies.add(ClientBuilderUtil.validateAndGetRetryPolicy(retryPolicy, retryOptions, new RetryPolicy()));
         policies.add(new AddDatePolicy());
-        if (azureKeyCredential != null) {
-            policies.add(new AzureKeyCredentialPolicy("api-key", azureKeyCredential));
+        policies.add(new CookiePolicy());
+        if (keyCredential != null) {
+            KeyCredentialPolicy keyCredentialPolicy;
+            // Connect to OpenAI service needs to have endpoints start with "https://api.openai.com/".
+            // By default, if user doesn't provide endpoint, it will redirect to create an Azure OpenAI service.
```

Comment:
If user doesn't provide endpoint, it should redirect to OpenAI and not Azure OpenAI.